Packages/com.unity.inputsystem/InputSystem/Controls/TouchControl.cs

```diff
@@ -27,42 +28,67 @@ public class TouchControl : InputControl<TouchState>
         public Vector2Control delta { get; private set; }
         public AxisControl pressure { get; private set; }
         public Vector2Control radius { get; private set; }
-        public PointerPhaseControl phase { get; private set; }
+        public InputControl<TouchPhase> phase { get; private set; }
```

Comment:
Why not TouchPhaseControl?

Reply:
Could go either way. My intention here was to hide the fact that it uses `TouchPhaseControl` as that control and `TouchButtonControl` are born out of implementation concerns. API-wise they don't really have much of a reason to exist.

Reply:
So what's the purpose of TouchPhaseControl?

Reply:
To not expose a raw `int` but rather a fully typed `TouchPase` value.

Thinking about it, think the fact the setup here is unexpected is probably reason enough not to do it this way. I'll go make it `TouchPhaseControl phase`.